pkg/server/server.go

```diff
@@ -558,6 +558,22 @@ func (server *Server) deleteApplication(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
+	active, err := server.clusterConfig.CheckActiveDeployments(organization, application)
+	if err != nil {
+		message := fmt.Sprintf("Could not check deployments for organization %s and application %s.  Error is %s", organization, application, err)
+		kiln.LogError.Printf(message)
+		internalError(message, w)
+		return
+	}
+
+	if active {
+		w.WriteHeader(http.StatusConflict)
+		w.Header().Set("Content-Type", "text/plain charset=utf-8")
```

Comment:
All of our API errors should be JSON.  I think `enrober` is already returning an error structure so you might want to look at that for the existing pattern.